sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/e-cert-utils.ts

```diff
@@ -0,0 +1,108 @@
+import { DisbursementSchedule, DisbursementValue } from "@sims/sims-db";
+import { E2EDataSources } from "@sims/test-utils";
+import { In } from "typeorm";
+
+/**
+ * Load disbursement awards for further validations.
+ * The method {@link awardAssert} can be used in conjunction with this.
+ * @param dataSource application dataSource.
+ * @param disbursementId disbursement id to have the awards loaded.
+ * @param options load options.
+ * - `valueCode` award value code to be filtered.
+ * @returns disbursement awards.
+ */
+export async function loadAwardValues(
```

Comment:
I would that return type could be `{[valueCode]:DisbursementValue}` so that `awardAssert` can lookup a key value instead of find. The existing way looks good. Just a suggestion as I felt key value could be better.

Reply:
`loadAwardValues` can be used independently of the `awardAssert`, it can be used together as mentioned in the comment but not necessarily need to.
Considering that this is a E2E helper, please let me now if that is a strong desired to have it changed.